src/main/java/org/nibor/autolink/LinkSpan.java

```diff
@@ -21,4 +21,8 @@
      */
     int getEndIndex();
 
+    /**
+     * @return the found sequence
+     */
+    CharSequence sequence();
```

Comment:
- [ ] Please remove this part of the change, it's not related to the WWW change at all as far as I can tell. It should be in a separate PR. But I'm also not sure I want to include a method like this, because it means that `LinkSpan` is required to keep either a reference to the original input or make a copy of the substring.